docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
-This means it gets an additional bonus between 2.67% and 4%
-This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.)
+The validator must unjail itself as quickly as possible after being jailed. 
 
-## Distribution Schedule
+#### Double signing event
+ 
+If a validator signs more than one block at the same height then 5% of their tokens are slashed. 
+This could be the case if someone duplicates their validator and tries to double their voting power.
 
-**Tx fees** are ideally the main source of revenue for validators and delegators. However, due to the expectation of low early engagement numbers on the Cudos blockchain, we will also provide staking rewards to users.
+#### What happens to slashed tokens?
 
-* We will distribute a total of ~1,5 billion tokens as staking rewards in ~10 years.
+Slashed tokens are sent to the community pool.
 
-* The exact number of tokens is given by the formula 
+:::warning Slashing
+When a validator’s tokens are slashed by a certain percentage, so are its delegators.
+:::
 
-**f(x) = 1.8 x^2 - 53 x + 358** 
+#### How to unjail
 
-**where *x* is in years and *f(x)* is in millions of CUDOS.**
+To unjail, the validator must send a transaction from the jailed validator account via the `Cudos-noded CLI`. 
 
-* Hence the amount to be distributed in a given time interval is obtained by integrating the above function over that interval.
+```shell
+cudos-noded tx slashing unjail --from mykey [flags]
+```
 
-* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
+### Unbonding period
 
-:::tip Important
+If a validator is jailed or leaves the bonded validators set, it enters an **unbonding period**. A validator that has failed to get itself back into the bonded validators set *within 21 days* is said to be **unbonded**.
 
-Important note: Validators need to use the --commission flag in the CLI in order to withdraw their commission.
+:::caution 
+Slashing can still occur during the unbonding period.
 :::
 
-## Jailing/Slashing
+### Token holder staking
 
-* Tokens delegated to a bonded validator is slashed if there is evidence of the validator misbehaving.
+Anyone holding CUDOS tokens can delegate or stake their tokens to one or more **validator nodes** of their choice. Validators and Stakers then share rewards **based on the amounts they have contributed to the pool**.
 
-* This means that when a validator’s tokens are slashed by a certain percentage, so are it delegators’.
+Validator Operators can set their own Commission to charge Stakers.**Commission rates** can change *once every day* however, the **Maximum Commission Rate** cannot be exceeded. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
```

Comment:
Validators can change their commission rates at most once in 24 hours, and by at most their "max commision **change** rate". In addition, a validator's commision rate cannot exceed its "max commision rate". Neither the "max commision change rate" nor the "max commision rate" parameters can be changed once the validator node is set up.

Reply:
Done